.github/workflows/update_node_version.yml

```diff
@@ -17,7 +17,7 @@ jobs:
             token: ${{ secrets.FPAT }}
       - name: Create node-version
         run: |
-          curl https://nodejs.org/en/download | grep -oE 'Latest LTS Version<!-- -->: <strong>[0-9]+\.[0-9]+\.[0-9]+</strong>' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' > .node-version
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```

Comment:
@bhupendra9211 This is suppose to create just .node-version only. But this will completely install nvm node manager. We use [nodenv](https://github.com/nodenv/nodenv)